src/code_800ED38.c

```diff
@@ -123,12 +123,15 @@ void sub_800EE5C(s32 r0)
     ret = sub_800ECA4(r0);
     if(r0 != 0)
     {
-        if((u32)(ret->unk0 - 1) > 1)
-        {
-            file = sub_800F1C0(ret->unk0, ret->unk4);
+        switch (ret->animType - 1) {
+        case 1 - 1:
```

Comment:
Just case 0 and 1, right?

Reply:
Yes, but note the -1 in the switch argument. This is an enum, I'm fairly sure

Reply:
Gotcha, just wanted to make sure this was intentional and not debug purposes.